themes/wds_headless/inc/wp-graphql.php

```diff
@@ -120,4 +120,38 @@ function wds_graphql_allowed_fields( array $fields, string $model_name ) {
 	}
 	add_filter( 'graphql_allowed_fields_on_restricted_type', 'wds_graphql_allowed_fields', 10, 6 );
 
+	/**
+	 * Include users without published posts in SQL query
+	 *
```

Comment:
@donmhico I would like function doc blocks to have both `@author WebDevStudios` and `@since 1.0`